modules/ROOT/pages/bolt/message.adoc

```diff
@@ -742,16 +757,57 @@ Valid values for `api` and the corresponding API associations are:
 - `2` -- Implicit transaction
 - `3` -- Driver-level `execute_query()`
 
+*Detail messages:*
+
+No detail messages should be returned.
+
+*Valid summary messages:*
+
+** `SUCCESS`
+** `FAILURE`
+
 ==== Synopsis
 ----
-TELEMETRY {metadata}
+TELEMETRY api
+----
+
+.Example 1
+----
```

Comment:
```suggestion
[source, Bolt]
----
```
And for all the others. This styles the codeblocks nicer (a bit stupid that the style doesn't get applied on "anonymous" blocks -- I guess it was a decision to avoid inadvertently styling stuff that shouldn't).